### DISCUSSION TITLE
Save/Restore Activation Codes?

### DISCUSSION THREAD

**Discussion by nurbles** [audiamus/AaxAudioConverter#128] [Ideas]
Now that it is (almost?) impossible to get the Audible Manager installer for Windows, I'm concerned that I may totally lose the ability to use AaxAudioConverter (AAC) on my _next_ computer. I've found a list of codes in the Basic Settings dialog, but there are EIGHT of them listed (I think, it is difficult to count in the tiny list box) and I neither know which one (or more?) is being used.  Also, when I looked at the manual entry, it appears that I can only enter one code?

Anyway, since AAC already has the necessary information, it would be great if there was a way to have the code(s) exported to a file that AAC could import later, after being re-installed on a machine that does not have Audible Manager installed/activated. If you're concerned about protecting the files, AAC could encrypt them so that only it could use them later, though I'd prefer a text option in case the codes are needed for something else in the future.

Is there any chance this could be (or is) on the next set of features for the next update?

**Comment by audiamus** [comment]
If you had used different Audible accounts in the past, legacy Audible Manager would have accumulated the activation codes for all. AAX Audio Converter then tries them one by one until it succeeds. I will think about a way to implement a kind of "smart" export function.

_Independent of that, my impression is that Audible may phase out the AAX format and its activation codes altogether and go for AAXC exclusively. That's what the Audible mobile phone apps download and [Book Lib Connect](https://github.com/audiamus/BookLibConnect) does, too.  It's pure speculation from my side, of course, but I take the withdrawal of the Audible Win 10 app as a clue in that direction._
